processing/src/main/java/org/apache/druid/query/expression/TimestampExtractExprMacro.java

```diff
@@ -141,13 +141,13 @@ public ExprEval eval(final ObjectBinding bindings)
             return ExprEval.of(dateTime.year().get());
           case DECADE:
             // The year field divided by 10, See https://www.postgresql.org/docs/10/functions-datetime.html
-            return ExprEval.of(Math.floor(dateTime.year().get() / 10));
+            return ExprEval.of(dateTime.year().get() / 10);
```

Comment:
Not sure if you need to handle BC?
Can you add tests?
i.e. 
-- SELECT EXTRACT(CENTURY FROM DATE '0101-12-31 BC'); 
-- SELECT EXTRACT(CENTURY FROM DATE '0100-12-31 BC'); 
-- SELECT EXTRACT(MILLENNIUM FROM DATE '0001-12-31 BC'); 
-- SELECT EXTRACT(DECADE FROM DATE '0002-12-31 BC');
-- SELECT EXTRACT(DECADE FROM DATE '0011-01-01 BC'); 
-- SELECT EXTRACT(DECADE FROM DATE '0012-12-31 BC');

Reply:
someone collected statistics for computers back in 101 BC 😂 

I'll add tests for correctness though 👍

Reply:
^ Please verify against Postgresql

Reply:
I decided it might be better not to support BC.

Since in ISO format, the date needs to be specified like `-0100-12-31` to signify the year `0101-12-31 BC` This would make the code complicated for a use case that is likely never to happen 🤞